src/zos_files/zowe/zos_files_for_zowe_sdk/uss.py

```diff
@@ -117,26 +131,40 @@ def write(self, filepath_name, data, encoding=_ZOWE_FILES_DEFAULT_ENCODING):
         response_json = self.request_handler.perform_request("PUT", custom_args, expected_code=[204, 201])
         return response_json
 
-    def get_content(self, filepath_name):
-        """Retrieve the content of a filename. The complete path must be specified.
+    def get_content(self, filepath_name: str) -> dict:
+        """
+        Retrieve the content of a filename. The complete path must be specified.
+
+        Parameters
+        ----------
+        filepath_name: str
+            Path of the file
 
         Returns
         -------
-        json
+        dict
             A JSON with the contents of the specified USS file
         """
         custom_args = self._create_custom_request_arguments()
         custom_args["url"] = "{}fs{}".format(self._request_endpoint, filepath_name)
         response_json = self.request_handler.perform_request("GET", custom_args)
         return response_json
 
-    def get_content_streamed(self, file_path, binary=False):
-        """Retrieve the contents of a given USS file streamed.
+    def get_content_streamed(self, file_path: str, binary: bool = False) -> dict:
+        """
+        Retrieve the contents of a given USS file streamed.
+
+        Parameters
+        ----------
+        file_path: str
+            Path of the file
+        binary: bool
+            Specifies whether the contents are binary
 
         Returns
         -------
-        response
-            A response object from the requests library
+        dict
+            A JSON response of results of the operaiotn
```

Comment:
Line 167 - "operaiotn" -> "operation"

```suggestion
            A JSON response with results of the operation
```